app/posts/2020-01-01-divide-a-design-history-into-different-sections.md

```diff
@@ -48,21 +47,31 @@ Next, create a page that lists these related posts. You can do that by creating
      reverse: true
      size: 50
    permalink: "support-interface/{% if pagination.pageNumber > 0 %}page/{{ pagination.pageNumber + 1 }}{% endif %}/"
-   eleventyComputed:
-     eleventyNavigation:
-       key: "{{ title }}"
-       excerpt: "{{ description }}"
-       parent: home
    ---
    ```
 
    {% endraw %}
 
+   The label appearing after `date: collections.` needs to match the tag you added to the JSON file for the section. This tells the index page which set of posts to list.
```

Comment:
Bit more pedantic possible, but I also got confused by `date: collections.`. Also, does a collection with a dash in the name not need to use the bracketed notation, i.e. `collections['support-interface']. Not sure.

```suggestion
   The value for `pagination.data` should use the collection created by the tag you added to the JSON file for the section. This tells the index page which posts to list. For example, if your tag is `support-interface`, the value for `pagination.data` would be `collections.support-interface`.
```

Reply:
Thanks - your version is much better. I wasn't sure how to explain this easily. (Didn't help that I typoed `data` as `date` 🤦 ).

I just checked and hyphens are fine to use within the frontmatter - `data: collections.support-interface` works, and even spaces seem to work somehow! (I guess the front matter syntax is based on YAML?)